codegen/templates/endpoint.tmpl

```diff
@@ -125,18 +138,18 @@ func (w {{$workflow}}) Handle(
 	{{end}}
 
 	{{if and (eq $clientReqType "") (eq $clientResType "")}}
-	_, err := w.Clients.{{$clientName}}.{{$clientMethodName}}(ctx, nil)
+	respHeaders, err := w.Clients.{{$clientName}}.{{$clientMethodName}}(ctx, headers)
```

Comment:
We do not want to proxy headers here.

We need a new whitelist in the endpoint json config for which headers to propagate. This is different from `.Headers` on the `MethodSpec`.